### DISCUSSION TITLE
Remedial Guide for Installing Outline

### DISCUSSION THREAD

**Discussion by mweth** [outline/outline#4651] [Show and tell] [answered]
I made a guide for installing Outline.  It is for VERY basic users, but it may be helpful nonetheless if you are trying to get someone onboarded.  It is targeted to lawyers, but that is just marketing spin for me and it should be useful to everyone.

https://www.wfirm.com/creating-a-self-hosted-knowledge-base-for-your-law-firm-using-outline-wiki/

**Comment by mweth** [comment, marked as answer]
Closing because I probably posted this in the wrong place.

**Reply by tommoor** [reply]
Not at all, thanks for writing the guide!
